thirdeye/thirdeye-frontend/app/styles/shared/_styles.scss

```diff
@@ -180,3 +264,42 @@ body {
 .te-link--hide {
   display: none;
 }
+
+.te-button {
```

Comment:
We've added a few styles under` styles/components/button/scss`, can we try to reuse them if possible?